OutOfSchool/OutOfSchool.ElasticsearchData/ESWorkshopProvider.cs

```diff
@@ -58,15 +58,14 @@ private QueryContainer CreateQueryFromFilter(WorkshopFilterES filter)
                 return queryContainer;
             }
 
-            if (!string.IsNullOrEmpty(filter.SearchText))
+            if (!(string.IsNullOrEmpty(filter.SearchText) || string.IsNullOrWhiteSpace(filter.SearchText)))
             {
```

Comment:
you can leave this condition only
string.IsNullOrWhiteSpace(filter.SearchText) it checks whether a specified string is null, empty, or consists only of white-space characters.

Reply:
thank you!